packages/commonwealth/client/scripts/views/pages/DirectoryPage/DirectoryPage.tsx

```diff
@@ -47,7 +48,13 @@ const DirectoryPage = () => {
   });
 
   const handleCreateCommunity = () => {
-    navigate('/createCommunity/starter', {}, null);
+    navigate(
+      featureFlags.newCreateCommunity
+        ? '/createCommunity'
```

Comment:
Suggestion: how about we update routes based on actions like
```
Create community: `/community/create`
Update community: `/community/:id/update`
```

etc?

Reply:
I am not sure about this. Following RESTful approach on the frontend does not seem to be right approach for me as we operate on the views, not resources. Currently we have `/manage` page where you perform actions related to updating/editing community but not only. Thus `community/:id/update` would be not valid.